wsagent/che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/ssh/SshKeyProvider.java

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - API
+ *   SAP           - initial implementation
+ *******************************************************************************/
+package org.eclipse.che.git.impl.jgit.ssh;
+
+import org.eclipse.che.api.git.GitException;
+
+/**
+ * @author Igor Vinokur
+ */
+public interface SshKeyProvider {
```

Comment:
Yes, it is the same class. Common `SshKeyProvider` will be used when https://github.com/eclipse/che/pull/1305 will be merged